### DISCUSSION TITLE
Deprem bölgesindeki Mutfaklar/Depolar için Gıda Stok Yönetimi

### DISCUSSION THREAD

**Discussion by savasdenli** [acikkaynak/deprem-yardim-projesi#80] [Ideas]
Deprem bölgelerinde 7 mutfak ve bunların malzeme tedariğini sağlayan depolar kuruldu. Bir de bu depolara gıda tedariği yapan firmalar mevcut. Basit bir stok yönetim ihtiyaçları var. 
- Depoya gelen ürünlerin stoğa eklenmesi (irsaliye ile geldiyse irsaliye fotoğrafı ile başta manuel belki sonrasında fotoğraftan okunup eklenebilir/irsaliye yoksa depo daki elemanın manuel giriş yapabileceği)
- Depodan çıkan ürünlerin stoktan düşülmesi (mutfaklara gönderilen ürünler manuel düşülür)
- Mutfakların depodan malzeme isteği (ilk aşamada şart değil ama mutfaklardaki şefler/ekip için işleri kolaylaştırır)
- Gıda türüne göre stok rule ları belirlenip hangi seviyenin altına inince kritik stok olduğu belirtilir / kritik stoklar listelemede en üste alınır böylece tedarikçi firmalar kritik stok olan ürünleri önceliklendirerek gönderebilir.
Böyle bir ihtiyaç var şu an direk talep edildi benden. Mümkünse bir ekip toplayıp yapabilirsek iyi olur.

**Comment by ahmetcumhurarslan** [comment]
mobil ve backend konusunda yardımcı olabilirim.
